packages/editor/src/private-apis.js

```diff
@@ -39,6 +40,7 @@ const { store: interfaceStore, ...remainingInterfaceApis } = interfaceApis;
 export const privateApis = {};
 lock( privateApis, {
 	CreateTemplatePartModal,
+	MediaEdit,
```

Comment:
I did this mostly to have a way to e2e test this component and might be probably useful to avoid issues like [this](https://github.com/WordPress/gutenberg/pull/68964) in the future (correct me if I'm wrong @Mamaduka ).

I have detailed comments about the testing approach [here](https://github.com/WordPress/gutenberg/pull/73537/files#diff-704d07e1a6f4e09c8bef8797f2904b442a56779f83856e8d2c003d7fa37f7c98R1).